src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -86,36 +86,46 @@ const ioTask = ({ name }) => _.nth(-2, name.split('.'))
 const ioVariable = ({ name }) => _.nth(-1, name.split('.'))
 const ioType = ({ inputType, outputType }) => (inputType || outputType).match(/(.*?)\??$/)[1] // unify, and strip off trailing '?'
 
+
 const WorkflowIOTable = ({ which, inputsOutputs: data, config, errors, onChange, onSetDefaults, onBrowse, suggestions, readOnly }) => {
+  const [sort, setSort] = useState({ field: 'taskVariable', direction: 'asc' })
```

Comment:
I'm tempted to suggest that when one of these columns is chosen as the primary sort, the other column should automatically be used as a secondary sort (in the same direction as the primary sort). Such a change would need to be applied here and in `_.orderBy` below.